torch/_export/__init__.py

```diff
@@ -425,6 +429,86 @@ def _reorder_kwargs_by_names(arg_names: List[str], args: Tuple[Any], kwargs: Dic
     return OrderedDict({kw_name: kwargs[kw_name] for kw_name in arg_names[len(args):]})
 
 
+def save(
+    ep: ExportedProgram,
+    f: Union[str, pathlib.Path, io.BytesIO],
+    extra_files: Optional[Dict[str, Any]] = None,
+    opset_version: Optional[Dict[str, int]] = None,
+) -> None:
+    """
+    Saves a version of the given exported program for use in a separate process.
```

Comment:
Saves an `ExportedProgram` to a file-like object. (Maybe cut "for use in a separate process.")